tests/test_categories.py

```diff
@@ -114,3 +117,8 @@ def raise_error(param):
 
         session.query = mock.Mock(side_effect=raise_error)
         assert get_user_categories(session, 1) == []
+
+    @staticmethod
+    def test_event_status_ok(client):
```

Comment:
Does the test check creating a new event with category?
Please rename or add doc.